src/script/pages/app-about.ts

```diff
@@ -10,6 +12,19 @@ export class AppAbout extends LitElement {
     super();
   }
 
+  async firstUpdated() {
+    const search = new URLSearchParams(location.search);
+    const site = search.get("site");
+
+    if (site) {
+      const manifestTestresults = await testManifest(site);
+      console.log('manifest test results', manifestTestresults);
+
+      const swTestResults = await testServiceWorker(site);
```

Comment:
Shouldn't we run these in parallel? As it stands, this would block on manifest detection before kicking off the service worker detection.

Reply:
Yep, this part specifically is not meant to be final code, but just a demo of how to make the calls that David will need for the waiting / testing page. Sorry, could have made that more clear.

Reply:
Thats also why im just console.logging the results haha

Reply:
Ah, I missed that. Thanks for clarifying.